docs/_macros/version_cleanup.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+class: Builder
```

Comment:
This should be `class: Macro` if you move the method to `Macro` or `class: Processor` if you leave it in `Processor`.